examples/bsp_ext.py

```diff
@@ -0,0 +1,68 @@
+import os
+import os.path
+import ruamel.yaml
```

Comment:
Since not every user of this project actually needs the "change the BSP of an example" function, I suggest making this optional. If ruamel.yaml isn't installed, don't fail here, and bail out from `action_extensions` function without registering anything.